transform-plugins/src/main/java/co/cask/hydrator/plugin/CSVParser.java

```diff
@@ -62,6 +66,10 @@
   // Output Schema associated with transform output.
   private Schema outSchema;
 
+  static final Set<String> FORMATS = new HashSet<>(Arrays.asList("DEFAULT", "EXCEL", "MYSQL", "RFC4180",
```

Comment:
nit: `private static final Set<String> FORMATS = ImmutableSet.of("DEFAULT", ...)`